### PR TITLE
Update downstream CSV description to link to official doc

### PR DESCRIPTION
**Pull request by jotak**
This is a small update to the CSV description used for downstream, that we should probably have done already earlier: removes the mentions of our unofficial guides, replaced with a link to the official doc

### CODE REVIEW

File: config/descriptions/ocp.md [hunk 1]
```diff
@@ -8,13 +8,9 @@ The operator provides dashboards, metrics, and keeps flows accessible in a query
 
 [Loki](https://grafana.com/oss/loki/), from GrafanaLabs, is the backend that is used to store all collected flows. The NetObserv Operator does not install Loki directly, however we provide some guidance to help you there.
 
-For normal usage, we recommend two options:
+- For a persistent usage, refer to [the operator documentation](https://docs.openshift.com/container-platform/latest/networking/network_observability/installing-operators.html).
```
[the remark this answers is not in the document]
Comment (skrthomas) [reply config/descriptions/ocp.md:11]: +1

**Review by skrthomas — APPROVED** [review]

File: config/descriptions/ocp.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jotak) [reply config/descriptions/ocp.md:11]: ok - I was thinking it's more than "production", for instance a persistent staging environment should also refer to the doc - any other thoughts then?

[the remark this answers is not in the document]
Comment (jotak) [reply config/descriptions/ocp.md:11]: @skrthomas , wdyt, sounds good to you?

[the remark this answers is not in the document]
Comment (skrthomas) [reply config/descriptions/ocp.md:11]: That suggestion is good with me!

**Comment by jotak** [comment]
thanks @nathan-weinberg @skrthomas - I updated with your suggestion .. and also found another outdated stuff regarding exporters, still mentioned as experimental

File: bundle/manifests/netobserv-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -438,7 +438,7 @@ spec:
 
     - Kafka (`spec.deploymentModel: KAFKA` and `spec.kafka`): when enabled, integrates the flow collection pipeline with Kafka, by splitting ingestion from transformation (kube enrichment, derived metrics, ...). Kafka can provide better scalability, resiliency and high availability ([view more details](https://www.redhat.com/en/topics/integration/what-is-apache-kafka)). Assumes Kafka is already deployed and a topic is created.
 
-    - Exporters (`spec.exporters`, _experimental_) an optional list of exporters to which to send enriched flows. Currently only KAFKA is supported. This allows you to define any custom storage or processing that can read from Kafka. This feature is flagged as _experimental_ as it has not been thoroughly or stress tested yet, so use at your own risk.
+    - Exporters (`spec.exporters`) an optional list of exporters to which to send enriched flows. KAFKA and IPFIX exporters are supported. This allows you to define any custom storage or processing that can read from Kafka or using the IPFIX standard.
```
Comment (skrthomas) [inline bundle/manifests/netobserv-operator.clusterserviceversion.yaml:441]:
```suggestion
    - Exporters (`spec.exporters`) an optional list of exporters to which to send enriched flows. KAFKA and IPFIX exporters are supported. This allows you to define any custom storage or processing that can read from Kafka or use the IPFIX standard.
```

**Pull request merged**